src/main/resources/elastic/mapping.json

```diff
@@ -0,0 +1,19 @@
+{
+  "mappings": {
+    "{{docType}}": {
+      "properties": {
+        "@type": {"type": "keyword"},
+        "@id": {"type": "keyword"},
+        "_rev": {"type": "keyword"},
+        "_deprecated": {"type": "boolean"},
+        "_createdAt": {"type": "date"},
+        "_updatedAt": {"type": "date"},
+        "_createdBy": {"type": "keyword"},
+        "_updatedBy": {"type": "keyword"},
+        "_constrainedBy": {"type": "keyword"},
+        "_source": {"type": "text"}
```

Comment:
What about the `_original` field ?

Reply:
I renamed `original` to `_source` as I thought that would be more meaningful.

Reply:
But it isn't renamed right...it is still [here](https://github.com/wwajerowicz/nexus-kg/blob/a366ebb21a86dc131bf3ec323d8449b79b2e60cb/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/ElasticIndexer.scala#L71)

Reply:
Yes, I thought that I renamed it, but I only added `_` 😄